components/pkg-export-kubernetes/tests/KubernetesManifestTestBinds.yaml

```diff
@@ -0,0 +1,45 @@
+---
+## Secret for initial configuration.
+#apiVersion: v1
+#kind: Secret
+#metadata:
+#  name: user-toml-secret
+#type: Opaque
+#data:
+#  # Each configuration item needs to be encoded in base64.
+#  # Plain text content of the secret: "port = 4444"
+#  user.toml: cG9ydCA9IDQ0NDQ=
+#---
+apiVersion: habitat.sh/v1
+kind: Habitat
+metadata:
+  ## Name of the Habitat service.
+  name: hi-f84iq
+  labels:
+    habitat-name: hi
+spec:
+  ## Name of the Habitat service package exported as a Docker image.
+  image: core/hi
+  ## Number of desired instances.
+  count: 3
+  ## An object containing parameters that effects how the Habitat service
+  ## is executed.
+  service:
+    ## Habitat topology of the service.
+    topology: standalone
+    ## Habitat service group name, a logical grouping of services
+    ## with the same package.
+    group: group1
+    ## Name of the configuration secret. Edit the Kubernetes Secret
+    ## at the top.
+    configSecretName: alphabravocharlie
```

Comment:
Again as above, do we need to have the secret name, this would not work if the secret is not created.

Reply:
we don't need it, no but this is just a sample configuration for testing (i-e if we specify the secret to the exporter, does it correctly end up in the generated manifest or not. This is not meant for deploying in the cluster.

Reply:
I mean, this is not how we want the file to end up looking. If I understand correctly this is the resulting file, right?
Then it should have the correctly generated manifest file. If the secret is commented out but the `configSecretName` is there, that should not be a valid file.